app/views/contexts/email_form_context.py

```diff
@@ -1,7 +1,13 @@
+from typing import Any
+
 from flask import url_for
 
+from app.forms.email_form import EmailForm
+
 
-def build_confirmation_email_form_context(email_confirmation_form):
+def build_confirmation_email_form_context(
+    email_confirmation_form: EmailForm,
+) -> dict[str, Any]:
```

Comment:
I would try to avoid using `Any` at all cost and figure out the type whenever possible, this will be raised by other devs reviewing this PR.

Reply:
I have updated it to specify top-level parameters.. but here I believe Any will be required. Specifying Sequence/ Stringfield structure will be a little complex. Please correct me if you disagree. Thanks